src/collections/pages/pages.11tydata.js

```diff
@@ -0,0 +1,55 @@
+"use strict";
+
+const slugify = require("slugify");
+
+module.exports = {
+    layout: "layouts/page.njk",
+    eleventyComputed: {
+        /* Determine the language of this item based on the language code in the file path. */
+        lang: data => data.page.filePathStem.replace(/\/collections\/pages\/([A-Za-z-]*)\/[a-z0-9-]*/g, "$1"),
+        /* Set the translationKey, used for populating the language switcher, to the file slug. */
+        translationKey: data => {
+            const lang = data.page.filePathStem.replace(/\/collections\/pages\/([A-Za-z-]*)\/[a-z0-9-]*/g, "$1");
+
+            if (data.page.fileSlug === lang) {
+                return "index";
+            }
+
+            return data.page.fileSlug;
+        },
+        eleventyNavigation: data => {
+            /* If this page has an `order` attribute, create an Eleventy Navigation object for it. */
+            if (data.order) {
+                return {
+                    parent: data.lang,
+                    order: data.order,
+                    /* If a key is set, use that for the navigation item label; otherwise use the page title. */
+                    key: data.hasOwnProperty("key") ? data.key : data.title
+                };
+            }
+            return false;
+        },
+        /* Build a permalink using the post title and language key. */
+        permalink: data => {
```

Comment:
This function is also nearly totally duplicated in `posts.11tydata.js`, so it would be good to reduce duplication where it's practical.

If an author wishes to configure pages and posts separately they will be able to do so deliberately, i.e. we'll encourage a DRY config and if they want to make it WET that'll be up to them.

Reply:
Resolved in 0975e69.